src/app/mate/create/components/CalendarDatePicker/CalendarDatePicker.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+
+import "react-datepicker/dist/react-datepicker.css";
+import { ko } from "date-fns/locale";
+
+import * as S from "./CalendarDatePicker.styles";
+
+interface CalendarDatePickerProps {
+  startDate: Date | null;
+  setStartDate: (date: Date | null) => void;
+}
+
+interface CustomInputProps {
+  value?: string;
+  onClick?: () => void;
+}
+
+const CalendarDatePicker = ({ startDate, setStartDate }: CalendarDatePickerProps) => {
+  const CustomInput = React.forwardRef(({ value, onClick }, ref) => (
+    <S.CustomInput
+      ref={ref}
+      onClick={onClick}
+      $isSelected={!!startDate}
+    >
+      {value || "모임 날짜를 선택해주세요."}
+    </S.CustomInput>
+  ));
```

Comment:
[ P2 ]

여기 부분에서 Props 지정이 빠진것같아요!
```suggestion
const CalendarDatePicker = ({ startDate, setStartDate }: CalendarDatePickerProps) => {
  const CustomInput = React.forwardRef<HTMLButtonElement, CustomInputProps>(
    ({ value, onClick }, ref) => (
      <S.CustomInput
        ref={ref}
        onClick={onClick}
        $isSelected={!!startDate}
      >
        {value || "모임 날짜를 선택해주세요."}
      </S.CustomInput>
    ),
  );
```

Reply:
앗 props를 빼먹었군요 ㅠㅠ 추가해두겠습니다 !!